templates/ssm/HelloWorldPython.ssm.yaml

```diff
@@ -0,0 +1,44 @@
+# Hello world using Python
```

Comment:
nit: we have the `.ssm.yaml` and `.asl.yaml` suffixes, so the `ssm/` subdirectory is redundant.

Reply:
removed subdirectory